<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageManagerForm</class>
 <widget class="QWidget" name="ImageManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="imageListGroupbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Avalible Images</string>
   </property>
   <widget class="QListWidget" name="imageList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addAnotherImageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Add More Images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBackgroundButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Set As Background</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="currentImageGroupbox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>381</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Show Image Settings</string>
   </property>
   <widget class="QLabel" name="imageRefLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageRefLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Position</string>
    </property>
    <widget class="QRadioButton" name="radioButtonL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonM">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonR">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Position</string>
    </property>
    <widget class="QLineEdit" name="xPosLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>xPos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>yPos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yPosLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Or Specific your Position:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="behindCharactersLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="whatsThis">
     <string>name list sperate by ','</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Behind:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(Image behind this characters)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyImageChange">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>&amp;Add Step</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+A</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>279</width>
      <height>28</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="changeImageRefButton">
     <property name="text">
      <string>Rename</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeSelectedImageButton">
     <property name="text">
      <string>&amp;Delete</string>
     </property>
     <property name="shortcut">
      <string notr="true">Ctrl+D</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbHide">
     <property name="text">
      <string>Hide</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="xScaleLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>xScale:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addScaledImageButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AddScaled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transitions:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="effectsComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="maxCount">
     <number>1024</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>ATL</string>
    </property>
    <widget class="QComboBox" name="ATLsComboBox5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QComboBox" name="ATLsComboBox1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QComboBox" name="ATLsComboBox3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ATLsComboBox6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ATLsComboBox2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QComboBox" name="ATLsComboBox4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxCount">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="ATLEdit4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="ALTLoopChkBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>imageRefLabel</zorder>
   <zorder>imageRefLineEdit</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>label_2</zorder>
   <zorder>behindCharactersLineEdit</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>applyImageChange</zorder>
   <zorder>textEdit</zorder>
   <zorder>splitter</zorder>
   <zorder>xScaleLineEdit</zorder>
   <zorder>label_5</zorder>
   <zorder>addScaledImageButton</zorder>
   <zorder>label</zorder>
   <zorder>effectsComboBox</zorder>
  </widget>
  <zorder>currentImageGroupbox</zorder>
  <zorder>imageListGroupbox</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
